provider/server/exec_test.go

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and IronCore contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package server_test
+
+import (
+	"context"
+	"net/url"
+
+	iri "github.com/ironcore-dev/ironcore/iri/apis/machine/v1alpha1"
+	irimeta "github.com/ironcore-dev/ironcore/iri/apis/meta/v1alpha1"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Exec", func() {
+
+	It("should return an exec-url with a token", func(ctx SpecContext) {
+		By("creating the test machine")
+		resCreate, err := srv.CreateMachine(context.TODO(), &iri.CreateMachineRequest{
+			Machine: &iri.Machine{
+				Metadata: &irimeta.ObjectMetadata{},
+				Spec: &iri.MachineSpec{
+					Class: "x3-xlarge",
```

Comment:
Optional: `x3-xlarge` is used in multiple places. So, it can be set as a variable and used in all the places.

Reply:
Done, thanks.